src/open_inwoner/search/views.py

```diff
@@ -63,7 +63,7 @@ def search(self, form):
         # update form fields with choices
         for facet in results.facets:
             if facet.name in form.fields:
-                form.fields[facet.name].choices = facet.choices()
+                form.fields[facet.name].choices = facet.total_choices()
```

Comment:
the requirement is to show (0) if the total is zero, so the UX communication about the results is more clear - is is possible to add that?

<img width="501" alt="Screenshot 2023-09-25 at 11 19 10" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/381fdb4a-55ff-4294-832a-cc467c86d0f4">

Reply:
Of course. I appended a change to the PR.